app/bundles/SmsBundle/Translations/en_US/messages.ini

```diff
@@ -13,6 +13,8 @@ mautic.sms.config.form.sms.password="Auth Token"
 mautic.sms.config.form.sms.password.tooltip="Twilio Auth Token"
 mautic.sms.config.form.sms.sending_phone_number="Sending Phone Number"
 mautic.sms.config.form.sms.sending_phone_number.tooltip="The phone number given by your provider that you use to send and receive Text Message messages."
+mautic.sms.config.form.sms.disable_trackable_urls="Disable trackable urls"
+mautic.sms.config.form.sms.disable_trackable_urls.tooltip="This option disable trackable Urls and your SMS click stats will not work."
```

Comment:
Please be consistent, SMS is in capitals. URL's should too. _This option disables_

.

Reply:
Fixed